src/Toml/Bi/Map.hs

```diff
@@ -125,12 +124,14 @@ data BiMap e a b = BiMap
 instance Cat.Category (BiMap e) where
     id :: BiMap e a a
     id = BiMap Right Right
+    {-# INLINE id #-}
 
     (.) :: BiMap e b c -> BiMap e a b -> BiMap e a c
     bc . ab = BiMap
         { forward  =  forward ab >=>  forward bc
         , backward = backward bc >=> backward ab
         }
+    {-# INLINE (.) #-}
 
 -- | Inverts bidirectional mapping.
 invert :: BiMap e a b -> BiMap e b a
```

Comment:
I think `invert`, `iso` and `prism` functions should have INLINE pragmas as well.